src/mmw/package.json

```diff
@@ -33,6 +33,7 @@
     "leaflet": "0.7.3",
     "leaflet-draw": "git://github.com/michaelguild13/Leaflet.draw#e2fe3a4",
     "leaflet.locatecontrol": "0.43.0",
+    "leaflet-plugins": "git://github.com/azavea/leaflet-plugins",
```

Comment:
The change I made only lets us access the Google map stuff via browserify. To submit this as a PR, I think I'd to make it work more generally and have a test case and some instructions on how to run browserify.